wasm-engines.md

```diff
@@ -0,0 +1,224 @@
+# Wasm Engines
+
+Binaryen and Wabt are two WebAssembly Engines which also provides a set of tools
+for compiling and decompiling wasm binary files. Both are official WebAssembly Projects.
+
+# Binaryen
+
+## Getting and compiling binaryen
+
+Install development tools (`build-essential`) `cmake` and `make`:
```

Comment:
This step shouldn't be necessary for OS X. Maybe we can just assume that the user already has `cmake` installed? And add a footnote or something with instructions to install it.